src/apis/quiz/quiz.api.ts

```diff
@@ -0,0 +1,11 @@
+import client from '@/apis/axios';
+
+export const quizRequests = async () => {
+  const { data } = await client.get('',{
```

Comment:
빈 문자열보다는 이렇게 표현하는게 어떨까요?
```suggestion
  const { data } = await client.get('/' ,{
```